lib/ruby_lsp/requests/support/rubocop_runner.rb

```diff
@@ -61,6 +62,11 @@ def run(path, contents)
           super([path])
         rescue RuboCop::Runner::InfiniteCorrectionLoop => error
           raise Formatting::Error, error.message
+        rescue RuboCop::ErrorWithAnalyzedFileLocation => error
+          line = ":#{error.line}" if error.line
+          column = ":#{error.column}" if error.column
+          location = "#{path}#{line}#{column}"
+          raise Formatting::Error, "An error occurred while #{error.cop.name} cop was inspecting #{location}"
```

Comment:
This matches the message generated by RuboCop on the command line:
https://github.com/rubocop/rubocop/blob/5be2df73ec57e8a7615461b0006a3ab70cb16999/lib/rubocop/cop/team.rb#L247

Reply:
UX question: since this error will indicate an issue in a cop's implementation, which
- Not every developer will know how to fix
- Will require in depth debugging of RuboCop separately
Would it be enough to just mention which cop failed without fully reconstructing the location?

Reply:
I thought about that, but might make the message overly long. It's mentioned in the default bug report template: https://github.com/rubocop/rubocop/issues/new?assignees=&labels=&template=bug_report.md